text/profiles/0212-profiling-vision.md

```diff
@@ -0,0 +1,96 @@
+
+Currently Logs, Metrics, and Traces are widely accepted as the main “pillars” of observability, each providing a different set of data from which a user can query to answer questions about their system/application. However, to limit observability, arbitrarily, to three pillars does a disservice to main goal of observability.
+
+Profiling data can help further this goal by answering certain questions about a system or application which logs, metrics, and traces are less equipped to answer. We aim to facilitate implementations capable of best-in-class support for collecting , processing, and transporting this profiling data.
+
+Our goals for profiling align with those of OpenTelemetry as a whole:
+
+- **Profiling should be easy**: the nature of profiling offers fast time-to-value by often being able to optionally drop in a  minimal amount of code and instantly have details about application resource utilization
+- **Profiling should be universal**: currently profiling is slightly different across different languages, but with a little effort  the representation of profiling data can be standardized in a way where not only are languages consistent, but profiling data itself is also consistent with the other observability signals as well
+- **Profiling should be vendor neutral**: From one profiling agent, users should be able to send data to whichever vendor they like (or no vendor at all) and interoperate with other OSS projects
+
+## Current State of Profilers
+
+As it currently stands, the method for collecting profiles for an application and the format of the profiles collected varies greatly depending on several factors such as:
+
+- Language (and language runtime)
+- Profiler Type
+- Data type being profiled (i.e. cpu, memory, etc)
+- Availability or utilization of symbolic information
+
+A fairly comprehensive taxonomy of various profiling formats can be found on the [profilerpedia website](https://profilerpedia.markhansen.co.nz/formats/).
+
+As a result of this variation, the tooling and collection of profiling data lacks in exactly the areas in which OpenTelemetry has built as its core engineering values:
+
+- Profiling currently lacks compatibility: Each vendor, open source project, and language has different ways of collecting, sending, and storing profiling data and often with no regard to linking to other signals
+- Profiling currently lacks consistency: Currently profiling agents and formats can change arbitrarily with no unified criteria for how to take end-users into account
+
+## Making Profiling Compatible with other Signals
+
+Profiles are particularly useful in the context of other signals. For example, having a profile for a particular “slow” span in a trace yields more actionable information than simply knowing that the span was slow. The nature of profiling also provides the potential for a more “hands-off” approach to adding it to code compared to manual instrumentation needed for other signals.
```

Comment:
>The nature of profiling also provides the potential for a more “hands-off” approach to adding it to code compared to manual instrumentation needed for other signals.

I am not sure why this sentence is added to this particular section. It seems irrelevant and the comparison is also unclear. Why compare with manual instrumentation and not to autoinstrumentation of other signals which is also often hands-off?

Reply:
yeah true. The idea here was to say:
1. Linking tracing to other signals is very useful
2. Linking tracingto other signals is _easy_ (since it isn't a very manual process) 

Don't feel super strongly about that second part though we could reword or remove

Reply:
It it is not entirely true that manual instrumentation is needed for other signals to capture trace context. For example logging SDKs in some languages can also automatically capture the current trace context and emit trace id/span id in the logs without any manual instrumentation needed.

>Linking to other signals is easy

It is probably useful to say this ^^^ directly instead of bringing the argument that it is _easier_ for profiling than for other signals.

Reply:
We go more into the details about the process of linking to other signals later so I'll just remove this part